### DISCUSSION TITLE
New ESP config tools from ESPhome/Home Assistant

### DISCUSSION THREAD

**Discussion by uSlackr** [thorrak/fermentrack#646] [Ideas]
This might be worth watching - easier path to device configuration & deployment:

https://www.home-assistant.io/blog/2021/06/16/power-up-your-esp-projects/

**Comment by thorrak** [comment]
Ok now this is neat.

**Comment by thorrak** [comment]
I’m thinking that it makes sense to build this out and implement it at BrewFlasher.com as a web-based BrewFlasher alternative. Fermentrack retains the standard workflow, but links out to BrewFlasher.com with notes about specifically where the device needs to be connected.
